spring-webflux/src/main/java/org/springframework/web/reactive/socket/adapter/JettyWebSocketSession.java

```diff
@@ -36,13 +43,26 @@
 
 /**
  * Spring {@link WebSocketSession} implementation that adapts to a Jetty
- * WebSocket {@link org.eclipse.jetty.websocket.api.Session}.
+ * WebSocket {@link Session}.
  *
  * @author Violeta Georgieva
  * @author Rossen Stoyanchev
  * @since 5.0
  */
-public class JettyWebSocketSession extends AbstractListenerWebSocketSession<Session> {
+@SuppressWarnings("NullAway")
+public class JettyWebSocketSession extends AbstractWebSocketSession<Session> {
```

Comment:
@lachlan-roberts 
can you look at the websocket part of these review

Reply:
The JettyWebSocket API uses a demand model instead of suspend/resume.

The `AbstractListenerWebSocketSession` requires us to implement `suspendReceiving()` and `resumeReceiving()` so we're better off not extending that class.

Reply:
Ah yes! Suspend/resume is intrinsically racy and best avoided.